lib/main.dart

```diff
@@ -16,6 +16,7 @@
  */
 
 import 'package:flutter/material.dart';
+import 'package:ubuntu_wsl_splash/utils/win32utils.dart';
```

Comment:
prefer sorted imports (`directives_ordering`)

Reply:
Added to analysis_options.yaml.